keyboards/dz60/dz60.h

```diff
@@ -708,3 +708,30 @@
     { k30,  k31,   k32,   k33,  k34,   k35,   k36,  k37,   k38,   k39,   k3a,  k3b,  k3c,   k3d,   k3e   }, \
     { k40,  k41,   KC_NO, k43,  KC_NO, KC_NO, k46,  KC_NO, KC_NO, KC_NO, k4a,  k4b,  k4c,   k4d,   k4e   }  \
 }
+
+/* LAYOUT_gk64
```

Comment:
This is actually [one of QMK's Community Layouts](https://github.com/qmk/qmk_firmware/tree/8af72000851dbe2c50a66bf2b9452011cbfe835a/layouts/default/64_ansi).

```suggestion
/* LAYOUT_64_ansi
```

Reference: https://docs.qmk.fm/#/feature_layouts